commit 7aaf2a24260cabbfc04791291160a4100e550b5a
Author: Romain Bignon <romain@budget-insight.com>
Date:   Tue May 5 10:42:49 2015 +0200

    ImlibLoadError has been renamed to Imlib_Load_Error in latest version of imlib2

diff --git a/src/im/account.cpp b/src/im/account.cpp
index efde142..4ac5163 100644
--- a/src/im/account.cpp
+++ b/src/im/account.cpp
@@ -275,7 +275,7 @@ void Account::setBuddyIcon(string filename)
 				else
 				{
 					char** prpl_formats = g_strsplit(prplinfo->icon_spec.format,",",0);
-					ImlibLoadError err = IMLIB_LOAD_ERROR_UNKNOWN;
+					Imlib_Load_Error err = IMLIB_LOAD_ERROR_UNKNOWN;
 
 					close(temp_fd);
 					/* Try to encode in a supported format. */
